web/app/store/AllAttributesAliases.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2016 Anton Tananaev (anton.tananaev@gmail.com)
+ * Copyright 2016 Andrey Kunitsyn (abyss@fox5.ru)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+Ext.define('Traccar.store.AllAttributesAliases', {
+    extend: 'Ext.data.Store',
+    model: 'Traccar.model.AttributeAlias',
+
+    proxy: {
+        type: 'rest',
+        url: 'api/devices/aliases',
```

Comment:
I thought about it too. Should we mark it somehow that it is aliases for attributes? I mean can we have aliases for something else in feature?